pages/home.py

```diff
@@ -75,14 +79,17 @@
 
 @callback(
     Output(MyStores.input_data.value, "data"),
+    Output("url", "search", allow_duplicate=True),
```

Comment:
this also should be from ElementsId an not a string

Reply:
This will be resolved.